### DISCUSSION TITLE
What are the differences with Askama?

### DISCUSSION THREAD

**Discussion by frederikhors** [askama-rs/askama#161] [Q&A] [answered]
Hi, I'm new to Rust.

I was wondering if this is a fork of Askama, what are the differences with Askama?

**Comment by GuillaumeGomez** [comment, marked as answer]
I wrote a blog post about it [here](https://blog.guillaume-gomez.fr/articles/2024-07-31+docs.rs+switching+jinja+template+framework+from+tera+to+rinja).
